### DISCUSSION TITLE
List of installed OS-patches?

### DISCUSSION THREAD

**Discussion by ejgh-oe** [Security-Onion-Solutions/securityonion#12380] [2.4] [answered]
### Version

2.4.40

### Installation Method

Security Onion ISO image

### Description

other (please provide detail below)

### Installation Type

Distributed

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

8

### RAM

32

### Storage for /

500

### Storage for /nsm

500

### Network Traffic Collection

span port

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

Hi,
In our installation SecurityOnion is configured to automatically install OS-Updates. How can I find out which OS-Updates were installed and when they were installed?

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by TOoSmOotH** [comment, marked as answer]
You can view all update history using the dnf command:

`sudo dnf history`

This should output a bunch of IDs and the command that was run. I grabbed mine from today:

`sudo dnf history info 25`
```
Transaction ID : 25
Begin time     : Tue 20 Feb 2024 10:23:16 AM UTC
Begin rpmdb    : 10321ba102684b4eb2257f97da433c135ddefa71a87ee49ab3f1104c6af7dd48
End time       : Tue 20 Feb 2024 10:23:34 AM UTC (18 seconds)
End rpmdb      : b4ffb10f847f9c47cfb06277fbb6866c63dd5739fb8e4643d176e04d3dc0ccd5
User           : System <unset>
Return-Code    : Success
Releasever     : 9
Command Line   : --quiet -y upgrade
Comment        :
Packages Altered:
    Install       kernel-uek-5.15.0-203.146.5.1.el9uek.x86_64         @securityonion
    Install       kernel-uek-core-5.15.0-203.146.5.1.el9uek.x86_64    @securityonion
    Install       kernel-uek-modules-5.15.0-203.146.5.1.el9uek.x86_64 @securityonion
    Reason Change NetworkManager-1:1.44.0-4.0.1.el9_3.x86_64          @securityonion
    Removed       kernel-uek-5.15.0-103.114.4.el9uek.x86_64           @@System
    Removed       kernel-uek-core-5.15.0-103.114.4.el9uek.x86_64      @@System
    Removed       kernel-uek-modules-5.15.0-103.114.4.el9uek.x86_64   @@System```

**Comment by ejgh-oe** [comment]
Thanks alot :-)
